public/app/features/alerting/unified/components/export/GrafanaMuteTimingsExporter.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import { LoadingPlaceholder } from '@grafana/ui';
+
+import { alertRuleApi } from '../../api/alertRuleApi';
+
+import { FileExportPreview } from './FileExportPreview';
+import { GrafanaExportDrawer } from './GrafanaExportDrawer';
+import { allGrafanaExportProviders, ExportFormats } from './providers';
+interface MuteTimingsExporterPreviewProps {
+  exportFormat: ExportFormats;
+  onClose: () => void;
+}
+
+const GrafanaMuteTimingsExporterPreview = ({ exportFormat, onClose }: MuteTimingsExporterPreviewProps) => {
+  const { currentData: muteTimingsDefinition = '', isFetching } = alertRuleApi.useExportMuteTimingsQuery({
+    format: exportFormat,
+  });
+  const downloadFileName = `mute-timings-${new Date().getTime()}`;
+
+  if (isFetching) {
+    return <LoadingPlaceholder text="Loading...." />;
+  }
+  return (
+    <FileExportPreview
+      format={exportFormat}
+      textDefinition={muteTimingsDefinition}
+      downloadFileName={downloadFileName}
+      onClose={onClose}
+    />
+  );
+};
+
+interface GrafanaMuteTimingExporterPreviewProps extends MuteTimingsExporterPreviewProps {
+  muteTimingName: string;
+}
+const GrafanaMuteTimingExporterPreview = ({
+  exportFormat,
+  onClose,
+  muteTimingName,
+}: GrafanaMuteTimingExporterPreviewProps) => {
+  const { currentData: muteTimingsDefinition = '', isFetching } = alertRuleApi.useExportMuteTimingQuery({
+    format: exportFormat,
+    muteTiming: muteTimingName,
+  });
+  const downloadFileName = `mute-timings-${new Date().getTime()}`;
```

Comment:
```suggestion
  const downloadFileName = `mute-timing-${new Date().getTime()}`;
```


Also, should we include the `muteTimingName` in the file name?